### PR TITLE
Respect AddPrinter, AddHtmlPrinter, fsi object

### PR DESCRIPTION
**Pull request by dsyme**
- Always inject `fsi.AddPrinter`, `fsi.AddHtmlPrinter` etc. into the programming model for literate scripts.  This is a part of the work in https://github.com/fsprojects/FSharp.Formatting/issues/534
  
 - By default download image content referred to in outputs when generating .tex and .ipynb

 - Remove the `api` command from the command line tool (`build` generalises it)

 - Add missing search.js

 - Add `(*** include-fsi-output **)`

 - Add `(*** include-fsi-merged-output **)`

- This repo had numerous in-repo templates with a lot of duplication between them, complexifying things.  Simplifying this out to just one `_template.html` suitable for building project documentation

### CODE REVIEW

**Comment by dsyme** [comment]
@baronfel Thanks. This PR contains a lot more work now if you'd like to take another look

**Comment by baronfel** [comment]
@dsyme so I see! That one was a drive-by nitpick, I'll try to tuck in and see if I can follow what's going on.

File: tests/FSharp.Literate.Tests/EvalTests.fs [hunk 1]
```diff
@@ -270,6 +263,63 @@ let ``Can include-it`` () =
...
+  html1 |> shouldContainText "<b>HELLO</b>"
+
+[<Test>]
+let ``Can include-html-output-even-without-hash-if-and-print-transformer-works-for-html-too`` () =
+  let content = """
+type Html = Html of string
+type Quack = Quack of string
+fsi.AddPrintTransformer(fun (Quack h) -> box (Html h))
+fsi.AddHtmlPrinter(fun (Html h) -> seq [], "<b>QUACK</b>")
+Quack "<b>HELLO</b>"
```
Comment (Krzysztof-Cieslak) [inline tests/FSharp.Literate.Tests/EvalTests.fs:299]: I think it should be 
```suggestion
Quack "HELLO"
```

as you wrap into `<b>` in the printer - so we want to test if printer is executed

Comment (dsyme) [reply tests/FSharp.Literate.Tests/EvalTests.fs:299]: thanks

**Pull request merged**